bdb_transaction_cli/utils.py

```diff
@@ -0,0 +1,36 @@
+import json
+
+import click
+
+
+class JsonParamType(click.ParamType):
+    """ A Click parameter class that loads JSON """
+    name = 'JSON'
+
+    def convert(self, value, param, ctx):
+        # No try-except in case `value is None`, as it is a valid value.
+        # Still we want `json.loads` to visibly fail.
+        if value is None:
+            return value
+        return json.loads(value)
+
+
+def json_argument(*args, **kwargs):
+    """ Decorator for a JSON command line argument """
+    kwargs['type'] = JsonParamType()
+    if 'help' in kwargs:
+        del kwargs['help']
```

Comment:
Is there a reason we remove the help? Why can't we just do something like `@click.argument(..., type=JSON_TYPE)`, where `JSON_TYPE` is an exported instance of `JsonParamType`?